### ISSUE TITLE
Graphic artifacts in the menu since r1236

### ISSUE THREAD

**Issue by Magus667** [emukidid/swiss-gc#642]
Since version 1236 I experience graphic artifacts in the menu, horizontal green bands that appear seconds after starting the console and that appear when scrolling through the menu. If I scroll a place they appear for a fraction of a second and disappear if I don't scroll. The problem does not appear in versions prior to 1236. In the latest version (1250) the artifacts are still present, but after about 15 seconds moving through the menu they disappear. Sorry my bad english.
![IMG-0883](https://user-images.githubusercontent.com/99774046/154176875-94895442-eca2-4918-8942-cd0d530ace7a.jpg)
![IMG-0884](https://user-images.githubusercontent.com/99774046/154176890-056b82a8-2f19-4783-8b74-50cb23a991f6.jpg)
![IMG-0885](https://user-images.githubusercontent.com/99774046/154176899-aaa8a576-1c9f-43b8-862f-a37dd24ff458.jpg)

**Comment by Diegorella** [comment]
you must provide more information about the console type.
component cable is used...
or The gcloloader may be misplaced...
It looks like there's an interference with your NGC's region memory.

**Comment by Magus667** [comment]
Hello, as I mentioned previously, in versions prior to 1236 this does not happen. The setup is as follows:

Gamecube NTSC-U DOL 001 Silver
GC Loader PnP fw 1.1.2 + SD Sandisk Extreme Pro 128gb
Gamecube component cable (Original Nintendo)
Composite AV cable (for audio)
GB Player US Black + EverDrive GBA X5 MINI
Memory card Niko 128Mb Slot A
SDCARD ADAPTER Slot B (generic) + SD Sandisk Ultra 16gb
Gamecube Wavebird Controller (Original Nintendo)
Original AC Adapter from Nintendo
Sony PVM 20M2MDU (480i, YUV, SMPTE)

The console is directly connected to the PVM monitor. The CR2032 battery in the console is dead and I have not replaced it. Gamecube games and gameboy player work perfectly. The console is set to 480i for component video and the IPL language has been changed to Spanish. I insist, this is something that I have only noticed from version 1236 onwards and in the current version (1250) it remains present but it lasts about 10-15 seconds since the swiss menu loads (cold boot or IGR).
